catwalk/models/language_model.py

```diff
@@ -642,8 +643,8 @@ def _run_greedy_until(
             if isinstance(untils, str):
                 untils = [untils]
             # if any of the stop phrases are single tokens we can use that for early termination
-            primary_until = None
-            for tokenized_until in tokenizer(untils)["input_ids"]:
+            primary_until = tokenizer.eos_token_id
+            for tokenized_until in tokenizer(untils, add_special_tokens=False)["input_ids"]:
```

Comment:
what's this about?

Reply:
The `add_special_tokeens=False` prevents the tokenizer from adding a `<bos>` token of sorts at the start of the string, which some tokenizers do, and would mess up the test that there's only one token (unfortunately, other tokenizers, like Llama/Mistral, also adds a "space" token when you tokenize "\n", so this is still not effective for those - will be improved in next iteration of model handling, more centrally)